doc/locale/ja/LC_MESSAGES/news.po

```diff
@@ -8,25 +8,53 @@ msgid ""
 msgstr ""
 "Project-Id-Version: groonga storage engine 1.0.1\n"
 "Report-Msgid-Bugs-To: \n"
-"PO-Revision-Date: 2019-08-30 17:40+0900\n"
+"POT-Creation-Date: \n"
+"PO-Revision-Date: 2022-07-19 16:40+0900\n"
 "Last-Translator: Masafumi Yokoyama <yokoyama@clear-code.com>\n"
 "Language-Team: Japanese\n"
 "Language: ja\n"
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
 "Plural-Forms: nplurals=1; plural=0;\n"
-"X-Generator: Poedit 2.1.1\n"
+"X-Generator: Poedit 3.1.1\n"
```

Comment:
@komainu8 

The version of `Poedit` is updated to 3.1.1 because I am using that version.
Is it acceptable change?